docs/components/truncate.md

```diff
@@ -121,10 +121,10 @@ The content to truncate.
 **Note:** when using KTruncate for truncating collection of HTML elements (not text), to ensure the best experience with the component, please pass elements of equal height in the default slot. The component will base its guess for the height of the element on the height of each individual sibling, so passing elements of equal height will result in a more accurate and consistent output.
 
 :::tip TIP
-The component is reactive to its dimensions changes. To see this in action, try resizing your browser window and notice the behavior of the example below.
+The component is reactive to its dimensions changes. To see this in action, try resizing the container below.
 :::
 
-<KCard>
+<KCard class="resizable-card">
```

Comment:
nit: can you set a min-width on the resizable container?
![image](https://github.com/Kong/kongponents/assets/2229946/f51fdf84-3ffc-48e5-9c0f-c99fba6a2785)